reporting.md

```diff
@@ -159,14 +159,14 @@ is as follows:
 ### Report using MAILTO
 
 ```
-_smtp_tlsrpt.mail.example.com. IN TXT \
+_smtp-tlsrpt.mail.example.com. IN TXT \
```

Comment:
As I read the spec, "mail.example.com" is the sending domain. But I agree this makes the example quite unclear; let's instead use "_smtp-tlsrpt.example.com" as the example. 

@mrisher fyi